k8s/ithc/common/family-public-law/fpl-case-service.yaml

```diff
@@ -14,15 +14,14 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: fpl-case-service
-    version: 1.8.0
+    version: 1.10.4
```

Comment:
Looks like we are currently on [1.10.5](https://github.com/hmcts/fpl-ccd-configuration/blob/master/charts/fpl-case-service/Chart.yaml)

Reply:
updated it